src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="#"
```

Comment:
The attribute `crossorigin` should have a value of `anonymous` to support CORS and prevent loading failures for the fonts. Please replace the `#` with `anonymous`.